basestation/base_station.py

```diff
@@ -65,7 +66,7 @@ def decorated_func(*args, **kwargs):
 
 
 class BaseStation:
-    def __init__(self, app_debug=False):
+    def __init__(self, app_debug=False, reuseport = config.reuseport):
```

Comment:
I don't see the value of reuseport used anywhere? Don't we want to toggle the socket configuration on line 108 with this value?

Reply:
thanks, fixed